features/bevragen/fields.feature

```diff
@@ -0,0 +1,271 @@
+    | fields              | naam.geslachtsnaam              |
+    Dan heeft de response een persoon met alleen de volgende 'naam' gegevens
+    | naam          | waarde      |
+    | geslachtsnaam | Kierkegaard |
+
+  Scenario: meerdere velden van een 'gegevensgroep' veld van een persoon wordt gevraagd
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                            |
+    | type                | RaadpleegMetBurgerservicenummer   |
+    | burgerservicenummer | 000000152                         |
+    | fields              | naam.geslachtsnaam,naam.voornamen |
+    Dan heeft de response een persoon met alleen de volgende 'naam' gegevens
+    | naam          | waarde         |
+    | geslachtsnaam | Kierkegaard    |
+    | voornamen     | Yvonne Johanna |
+
+  Scenario: velden van meerdere 'gegevensgroepen' van een persoon wordt gevraagd
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                            |
+    | type                | RaadpleegMetBurgerservicenummer   |
+    | burgerservicenummer | 000000152                         |
+    | fields              | naam.geslachtsnaam,geboorte.datum |
+    Dan heeft de response een persoon met alleen de volgende 'naam' gegevens
+    | naam          | waarde      |
+    | geslachtsnaam | Kierkegaard |
+    En heeft de persoon de volgende 'geboorte' gegevens
+    | naam              | waarde           |
+    | datum.type        | Datum            |
+    | datum.datum       | 1956-11-15       |
+    | datum.langFormaat | 15 november 1956 |
+
+  Scenario: een veld van een 'gegevensgroep collectie' veld van een persoon wordt gevraagd
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | partners.burgerservicenummer    |
+    Dan heeft de response een persoon met een 'partner' met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999992971 |
+
+Rule: Een veld waarvan het volledig pad deels overeenkomt met een opgegeven veld pad wordt niet geleverd
+
+  Scenario: het opgegeven pad is het volledig pad van het voornamen veld én een deel van het volledig pad van het voornamen veld van de partner/ouder/kind velden van de persoon
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | naam.voornamen                  |
+    Dan heeft de response een persoon met alleen de volgende 'naam' gegevens
+    | naam      | waarde         |
+    | voornamen | Yvonne Johanna |
+    
```

Comment:
Duurde even voordat ik deze doorhad :-). Ik begrijp dat dit pad voor een gerelateerde onvolledig is (omdat daar kinderen of partners of ouders voor moet staan), en daarom niet wordt geleverd.